auth/src/main/res/layout/welcome_back_password_prompt_layout.xml

```diff
@@ -1,63 +1,78 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
I tried to make just minimal changes. For some reason it seems that button resizing works only if the content of the layout is wrapped by Scrollview. If I use different layouts, the SIGN IN button doesn't go above the keyboard.